src/site/content/en/blog/optimize-cls/index.md

```diff
@@ -202,7 +199,7 @@ Some sites may find collapsing the slot initially can reduce layout shifts if th
   </figure>
 
   <figure class="w-figure">
-    <video controls loop muted class="w-screenshot" poster="video-top-ad-after.jpg">
+    <video controls loop muted class="w-screenshot" poster="{{ "image/tcFciHGuF3MxnTr1y5ue01OGLBn2/ZzGB6CsKF7cmiIcVLlCA.jpg" | imgix }}">
```

Comment:
Same as above

Reply:
Currently the video is broken/missing, gonna have to wait on #5050 before we can fix this.